chcemvediet/templates/main/homepage/main.html

```diff
@@ -15,7 +15,10 @@
 {% block content %}
   <div class="chv-scheme-inv-blue">
     <div class="container text-center">
-      {% include "main/homepage/search.html" %}
+      {% include "main/snippets/search.html" %}
+      <div class="chv-bellow-1 center-block">
+        {% include "main/homepage/texts/search_text.html" %}
```

Comment:
`center-block` tu vobec netreba, kedze centrujeme iba text. V predoslom dive to bolo preto, ze vo vnutri boli aj netextove veci, ktore sa centrovali.

`chv-bellow-1` bude imho praktickejsie odtial vyhodit a dat ho do `homepage/steps.html` ako `<div class="row chv-above-1">`. Teda, ze na medzera nebude pod textom, ale nad dalsim divom.

Reply:
V tom pripade bude div zbytocne, takze som ho vyhodil.